src/classes/STG_SettingsManager_CTRL.cls

```diff
@@ -106,6 +106,11 @@ public with sharing class STG_SettingsManager_CTRL extends STG_Panel {
         }
     }
 
+    /*******************************************************************************************************
+    * @description our exception object for validating allocation settings.
```

Comment:
Since this exception is more generic, I recommend a description such as "Custom exception used when logging validation exceptions during NPSP Settings load".